src/main/java/projectzero/core/UmlClass.java

```diff
@@ -17,4 +19,17 @@ public String getName() {
     public void setName(String name) {
         this.name = name;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        UmlClass umlClass = (UmlClass) o;
+        return Objects.equals(getName(), umlClass.getName());
```

Comment:
Why Objects.equals? Should we be testing that the strings equals? Something like name.equals(umlClass.getName()) or getName().equals(umlClass.getName()).

Reply:
This is the autogenerated equals method that idea creates using the fields.